test/scripts/base-host-check.sh

```diff
@@ -80,17 +80,20 @@ get_oscap_score() {
 }
 
 check_ca_cert() {
-    serial=$(jq -r '.blueprint.customizations.cacerts.pem_certs[0]' "${config}" | openssl x509 -noout -serial | cut -d= -f 2-)
+    serial=$(jq -r '.blueprint.customizations.cacerts.pem_certs[0]' "${config}" | openssl x509 -noout -serial | cut -d= -f 2- | tr '[:upper:]' '[:lower:]')
+    cn=$(jq -r '.blueprint.customizations.cacerts.pem_certs[0]' "${config}" | openssl x509 -noout -subject | sed -E 's/.*CN ?= ?//')
 
-    echo "📗 Checking CA cert anchor file"
+    echo "📗 Checking CA cert anchor file serial '${serial}'"
     if ! [ -e "/etc/pki/ca-trust/source/anchors/${serial}.pem" ]; then
-        echo "Anchor CA file does not exist"
+        echo "Anchor CA file does not exist, directory contents:"
+        find /etc/pki/ca-trust/source/anchors
         exit 1
     fi
 
-    echo "📗 Checking extracted CA cert file"
-    if ! [ -e "/etc/pki/ca-trust/source/extracted/pem/directory-hash/Test_CA_for_osbuild.pem.pem" ]; then
-        echo "Extracted CA file does not exist"
+    echo "📗 Checking extracted CA cert file named '${cn}'"
+    if ! grep -q "${cn}" /etc/pki/ca-trust/extracted/pem/tls-ca-bundle.pem; then
+        echo "Extracted CA file does not exist, directory contents:"
+        find /etc/pki/ca-trust/extracted/pem
```

Comment:
The grep-ing of `tls-ca-bundle.pem` makes me wonder if listing the content of `/etc/pki/ca-trust/extracted/pem` still makes sense? :thinking:

Reply:
So, judging from https://github.com/osbuild/osbuild-composer/pull/4487/files#diff-a0a3d509a4e8903c0bede204d82662223fa270c20360d5bf2dcd2f1d2b68bbcdR264, it should be probably
```shell
grep '^#' /etc/pki/ca-trust/extracted/pem/tls-ca-bundle.pem
```

Reply:
Thank you for the change. Please also amend the comment above it.